### DISCUSSION TITLE
Steps to add a new repo to gee-community

### DISCUSSION THREAD

**Discussion by giswqs** [gee-community/gee-community.github.io#2] [Announcements]
1. Create the repository elsewhere, such as under your personal account.
2. File a ticket [here](https://github.com/orgs/gee-community/discussions) asking to join the organization, including a link to the repository being added.
3. Accept the invitation to join the organization.
4. Use the GitHub UI to move the repository into the organization (you will need administrator/owner rights on the repository to do this). See [the GitHub documentation on repository transfer](https://help.github.com/en/github/administering-a-repository/transferring-a-repository) for details.
5. After it is moved, rename the repository as needed. Don't forget to update the documentation in the repository, too!

**Comment by lopezvoliver** [comment]
Hi, I would like to contribute this [repository](https://github.com/lopezvoliver/eetasks), but the link mentioned in step 2. above does not work for me (Page not found: 404 error). Please fix it so I can follow up.

**Comment by giswqs** [comment]
Thanks for pointing out the broken link. It has been fixed.

**Reply by lopezvoliver** [reply]
Hi @giswqs , could you take a look when you get a chance? Thanks 

https://github.com/orgs/gee-community/discussions/8
